openpype/plugins/publish/collect_otio_subset_resources.py

```diff
@@ -173,6 +173,34 @@ def process(self, instance):
             dirname, filename = os.path.split(media_ref.target_url)
             self.staging_dir = dirname
             if trimmed_duration < available_duration:
+                # first convert trimmed start to timecode with timeline fps
+                # and then convert to frames in media fps
+                timeline_fps = otio_clip.source_range.start_time.rate
+                self.log.debug("timeline_fps: {}".format(timeline_fps))
+                self.log.debug("media_fps: {}".format(media_fps))
+                if (
+                    instance.context.data["hostName"] == "resolve" and
```

Comment:
Should this logic be here if is "resolve" specific? Considering we'll be splitting all the parts into addons this can cause huge issues in future.